app/views/buyers/applications/new.html.erb

```diff
@@ -14,10 +14,12 @@
 
   <%= form.semantic_errors %>
   <%= form.inputs do %>
-    <%= form.input :plan, :as => :plan_selector,
-                          :collection => @plans, :include_blank => false,
-                          :label => ApplicationPlan.model_name.human,
-                          :default_plan => @plans.respond_to?(:default) ? @plans.default : @plans.first %>
```

Comment:
@mikz do you know how is `default` evaluated? Is this some kind of method? I don't see any property in the model that I can use in JS (`plan.default -> undefined`).

Reply:
https://github.com/3scale/porta/blob/6f74cf39b5bfef598ef24388b17fb98ad3618484/app/models/account/provider_methods.rb#L116-L120